backend/schema.js

```diff
@@ -62,13 +63,14 @@ type Show {
 }
 
 input ShowInput {
-    name: String!
+    name: String
```

Comment:
Why did you remove the required checks?

Reply:
To allow for updates to work properly. It was too test stuff out, I'll create a ShowUpdate type.